src/schema/objects/columns.yaml

```diff
@@ -302,13 +302,11 @@ notch:
   display_name: Notch frequencies
   description: |
     Frequencies used for the notch filter applied to the channel, in Hz.
+    If the low and high frequencies are supplied in addition to the center
+    frequency, it is RECOMMENDED that they take the form `[low, center, high]`,
```

Comment:
Why not required? 

Would another format be possible if the low and high frequencies are provided?

Reply:
Because it hasn't previously been required. We have another PR proposing to use comma-separated values for multiple channels in a single TSV column. My goal here is to accommodate existing behavior, not impose a new rule.

If the ephys community wants to adopt that, that's fine. I would suggest then that I simply relax the type here and leave the text unchanged. You or someone else who actually uses these data could lead an effort to get buy-in to have more specific requirements for three-value notch specifications.